dashboard/src/components/MainView/MainView.js

```diff
@@ -8,32 +8,39 @@ import PollutantDetails from "../Pollutants/PollutantDetails";
 import TopCountries from "../TopCountries/TopCountries";
 import { getCityData, getCountryData } from "../../api";
 
-
-const MainView = ({ allPollutants = [] }) => {
+const MainView = () => {
   const navigate = useNavigate();
   const [selectedPollutant, setSelectedPollutant] = useState("");
   const [searchValue, setSearchValue] = useState("");
   const [searchType, setSearchType] = useState("");
 
-
   const searchOptions = [
-    { value: '', label: 'Select search parameter' },
-    { value: 'city', label: 'City' },
-    { value: 'country', label: 'Country' },
+    { value: "", label: "Select search parameter" },
+    { value: "city", label: "City" },
+    { value: "country", label: "Country" },
   ];
 
-  const onSearchButtonClick = async () => {
+  const handleChange = (event) => {
+    setSearchType(event.target.value);
+  };
 
-    const countryResponse = await getCountryData("Mexico");
-    console.log(countryResponse);
-    const cityResponse = await getCityData("London");
-    console.log(cityResponse);
-    const result = await getCityData(searchedCity)
-      .then(response => response.results);
-    setCityData(result[0]); //getCityData will now return either an element of the results array (station w the highest # of recorded parameters)
-    //or the response as a whole if the results array is empty or missing - this needs to be adjusted as a result.
-  }
+  const onSearchButtonClick = async () => {
+    navigate("/detailed", {
+      state: {
+        searchedValue: searchValue,
+        searchedType: searchType,
+      },
+    });
 
+    // const countryResponse = await getCountryData("Mexico");
+    // console.log(countryResponse);
+    // const cityResponse = await getCityData("London");
+    // console.log(cityResponse);
+    // const result = await getCityData(searchedCity)
+    //   .then(response => response.results);
+    // setCityData(result[0]); //getCityData will now return either an element of the results array (station w the highest # of recorded parameters)
+    // //or the response as a whole if the results array is empty or missing - this needs to be adjusted as a result.
+  };
```

Comment:
We should think about deleting out this commented out code just so we don't forget to later

Reply:
once we're done with the implementation, we gotta clean up all the code.

Reply:
I agree, I think mainly one person with eslint and prettier enabled to follow the airbnb style guide should go through and make sure everything is compliant.